src/tailwind.css

```diff
@@ -154,6 +154,14 @@
     clear: both;
   }
 
+  .event-details h3 {
+    margin-top: 1rem;
+  }
+
+  .event-details p {
+    margin-top: 0.5rem;
+  }
+
```

Comment:
remove, let's not do 1 off customisations to wysiwyg. they should ideally only inherit base styles

Reply:
ok, updated